tests/actions.go

```diff
@@ -345,7 +353,7 @@ func (oi *OperatorConfig) OperatorHelmSetString(m map[string]string) string {
 func (oa *operatorActions) DeployOperator(info *OperatorConfig) error {
 	glog.Infof("deploying tidb-operator %s", info.ReleaseName)
 
-	if info.Tag != "e2e" {
+	if info.Tag != "e2e" && info.Tag != "stability" {
```

Comment:
Why do we take a snapshot of helm charts into the stability test image? 

Suppose the test is running indefinitely, I think it's better to make the repo url and tag configureable in order to test the head of specific branch or tag in each run, e.g. testing the head of `master` branch or `branch-1` of `https://github.com/pingcap/tidb-operator`

Reply:
ours develop branch is in fork repo,  this snapshot is used to test the feature which has not merged to master branch.

Reply:
How about configure the repo url like #499 ?

Reply:
0h, it's fine for me, we needed merge #499 before this pr.
I will review it right now.